openreview/tools.py

```diff
@@ -107,8 +121,23 @@ def get_profiles(client, ids_or_emails, with_publications=False):
             })))
 
     if with_publications:
+
+        baseurl_v1 = 'http://localhost:3000'
+        baseurl_v2 = 'http://localhost:3001'
+
+        if 'https://devapi' in client.baseurl:
+            baseurl_v1 = 'https://devapi.openreview.net'
+            baseurl_v2 = 'https://devapi2.openreview.net'
+        if 'https://api' in client.baseurl:
+            baseurl_v1 = 'https://api.openreview.net'
+            baseurl_v2 = 'https://api2.openreview.net'
+
+        client_v1 = openreview.Client(baseurl=baseurl_v1, token=client.token)
+        client_v2 = openreview.api.OpenReviewClient(baseurl=baseurl_v2, token=client.token)
         for profile in profiles:
-            profile.content['publications'] = list(iterget_notes(client, content={'authorids': profile.id}))
+            notes_v1 = list(iterget_notes(client_v1, content={'authorids': profile.id}))
+            notes_v2 = list(iterget_notes(client_v2, content={'authorids': profile.id}))
+            profile.content['publications'] = notes_v1 + notes_v2
```

Comment:
Not necessary, but you could speed this up like this:
```py
notes_v1 = concurrent_requests(lambda profile : list(iterget_notes(client_v1, content={'authorids': profile.id})), profiles)
for idx, publication in enumerate(notes_v1):
    profiles[idx].content['publications'] = publication

notes_v2 = concurrent_requests(lambda profile : list(iterget_notes(client_v2, content={'authorids': profile.id})), profiles)
for idx, publication in enumerate(notes_v2):
    if profiles[idx].content['publications']:
        profiles[idx].content['publications'] = profiles[idx].content['publications'] +  publication
    else:
        profiles[idx].content['publications'] = publication

```

Reply:
I don't think a user would have more 1k publications, let's keep it as it is now. Thanks for the shared code.

Reply:
Yes, I think it's OK. To explain the code a bit: This gets all the publications of the profiles in parallel, it does not go one by one for each profile. In other words, the `for profile in profiles` is removed. That's why it's faster :)

Reply:
ohhhh I see now, let me do the change now